recipes/tailwind/templates/config/tailwind.config.js

```diff
@@ -1,6 +1,8 @@
 // tailwind.config.js
 module.exports = {
-  content: ["./{src,app,pages}/**/*.{js,ts,jsx,tsx}"],
+  content: [
+    "./{src/pages,src/components,src/app,src,pages,components,app}/**/*.{js,ts,jsx,tsx,mdx}",
+  ],
```

Comment:
🍰 https://nextjs.org/docs/app/building-your-application/styling/tailwind-css#configuring-tailwind
```suggestion
  content: [
    './app/**/*.{js,ts,jsx,tsx,mdx}',
    './pages/**/*.{js,ts,jsx,tsx,mdx}',
    './components/**/*.{js,ts,jsx,tsx,mdx}',
    './src/**/*.{js,ts,jsx,tsx,mdx}',
  ],
```